source/templates/code_signing.slim

```diff
@@ -1,21 +1,25 @@
 .tab-code-signing[ng-controller="CodeSigningController as codeSigningCtrl"]
 	section.progress-indicator[progress-model="codeSigningCtrl.loadProgress"]
-	section.certificate-help[ng-if="codeSigningCtrl.shouldShowCodeSigningHints"]
-		ul
-			li.codesigndoc
-				div
-					h2 == data[:strings][:code_signing][:certificate_help][:codesigndoc][:title]
-					p == data[:strings][:code_signing][:certificate_help][:codesigndoc][:details]
-					code == data[:strings][:code_signing][:certificate_help][:codesigndoc][:code]
-			li.upload
-				div
-					h2 == data[:strings][:code_signing][:certificate_help][:upload][:title]
-					p == data[:strings][:code_signing][:certificate_help][:upload][:details]
-			li.add-step
-				div
-					h2 == data[:strings][:code_signing][:certificate_help][:add_step][:title]
-					p == data[:strings][:code_signing][:certificate_help][:add_step][:details]
-		button.hide[ng-click="codeSigningCtrl.codeSigningHintsDismissed()"] = svg("cross_slim")
+	section.certificate-help[ng-if="codeSigningCtrl.shouldShowSectionOfCodeSigningHints"]
+		.toggleabble-support-info
+			span.oneliner[ng-if="!codeSigningCtrl.shouldShowCodeSigningHints"]
+				span = svg("support")
+				span == data[:strings][:code_signing][:certificate_help][:title]
+			ul[ng-if="codeSigningCtrl.shouldShowCodeSigningHints" ng-class="{'oneliner': !codeSigningCtrl.shouldShowCodeSigningHints}"]
```

Comment:
The `ng-class` should be on the parent class, and should rather be called `open` (with the opposite condition). And that open should style the span.oneliner (h2, after changes above), as well as the ul.